.gitignore

```diff
@@ -2,3 +2,4 @@
 **/__debug*
 **/*_templ.go
 tmp/
+**/main
```

Comment:
What's this ignore for?

Reply:
I think you can ignore this, I'll revert it. I think this is something that my IDE added when I ignored a compiled version of the application.